programming-basics/fibonacci/test.js

```diff
@@ -0,0 +1,28 @@
+import fib from './main'
+
+describe('programming-basics[001]', () => {
+  it(`should calculate fibonacci of 0`, () => {
+    const result = fib(1)
+    expect(result).toBe(1)
+  })
+
+  it(`should calculate fibonacci of 1`, () => {
+    const result = fib(2)
+    expect(result).toBe(1)
+  })
+
+  it(`should calculate fibonacci of 10`, () => {
+    const result = fib(10)
+    expect(result).toBe(55)
+  })
+
+  it(`should calculate fibonacci of 100`, () => {
+    const result = fib(20)
+    expect(result).toBe(6765)
+  })
+
+  it(`should calculate fibonacci of -1`, () => {
```

Comment:
I'm not sure about this testcase, if we should handle negative numbers or if we should let it break...

I would like from @cassios  and @luisarthurbv what they think about it.